src/scripts/setup_test_data/setup_test_data.py

```diff
@@ -26,13 +27,15 @@ def get_synapse_client(ssm_parameter):
     Name=ssm_parameter,
     WithDecryption=True)
   syn = synapseclient.Synapse()
-  syn.login(authToken=token['Parameter']['Value'])
+  synapse_auth_token = token['Parameter']['Value']
+  os.environ['SYNAPSE_AUTH_TOKEN'] = synapse_auth_token
+  syn.login(authToken=synapse_auth_token, silent=True)
```

Comment:
Is there a reason you do both?  Unless you call `syn.login` later on, I don't think you need to have both `authToken=...` and `os.environ....`

Reply:
I think there was a version of synapseformation that required that I log in explicitly; I'm not recalling exactly now. I have to have a syn because I use that directly around the synapseformation call, and since create_synapse_resources takes a syn, I can just do away with the env var.

Reply:
Wait, I'm just realizing that what I wrote doesn't make sense.

Reply:
I have the latest version of synapseclient (2.5.0) but setting the env var only didn't work for me, so I removed that.